internal/proxy/impl.go

```diff
@@ -2753,11 +2753,18 @@ func (node *Proxy) HybridSearch(ctx context.Context, request *milvuspb.HybridSea
 	qt := &hybridSearchTask{
 		ctx:       ctx,
 		Condition: NewTaskCondition(ctx),
-		request:   request,
-		tr:        timerecord.NewTimeRecorder(method),
-		qc:        node.queryCoord,
-		node:      node,
-		lb:        node.lbPolicy,
+		HybridSearchRequest: &internalpb.HybridSearchRequest{
+			Base: commonpbutil.NewMsgBase(
+				commonpbutil.WithMsgType(commonpb.MsgType_Search),
+				commonpbutil.WithSourceID(paramtable.GetNodeID()),
+			),
+			ReqID: paramtable.GetNodeID(),
```

Comment:
should not be NodeID?

Reply:
> should not be NodeID?

search request also use nodeID. https://github.com/milvus-io/milvus/blob/a27c0e86fde05b4409bfa7130b62e0089d25bad0/internal/proxy/impl.go#L2618